.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+
+jobs:
+  test:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
You mention that `skip ci` does not work anymore, but this seems to indicate otherwise. If this line does not work/has no effect, I think we should remove it.

Reply:
This works on `push` events but not in PRs. In particular, it works (or should do so) when you push, e.g., a small change of the README directly to `master`. Hence, I would like to keep it.

Reply:
:+1: